src/modules/notary/index.ts

```diff
@@ -0,0 +1,137 @@
+import Ajv, { JTDSchemaType } from 'ajv/dist/jtd';
```

Comment:
Can you add to the tech spec the high-level operations that are taking place for the message decryption and validation? Help me and new engineers understand it.